src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
Note: This analyzer works with Properties, and even so, if there aren´t acessors, it not a property, it´s a Field and analyzer will not called. It´s not necessary check AcessorList == null. For a property, always has almost one acessor: get or set.